yb-voyager/cmd/templates/migration_assessment_report.template

```diff
@@ -274,8 +274,44 @@
 
         {{if and (ne .MigrationComplexity "NOT AVAILABLE") (ne (len .MigrationComplexityExplanation) 0)}}
             <h2>Migration Complexity Explanation</h2>
-            <p>{{ .MigrationComplexityExplanation }}</p>
-        {{end}}
+            {{- if .MigrationComplexityCategorySummary }}
```

Comment:
Q: what does `-` mean here?

Reply:
It helps to avoid extra whitespaces caused due to if conditions in the html
<img width="801" alt="image" src="https://github.com/user-attachments/assets/fd717976-8ca6-4343-9fbf-4847d243f264" />